.github/workflows/go-production.yml

```diff
@@ -186,7 +186,7 @@ jobs:
       - name: Build orb-agent
         shell: bash
         env:
-          IMAGE_NAME: ns1labs/orb-agent
+          IMAGE_NAME: orbcommunity/orb-agent
```

Comment:
just checking - is this supposed to be `orbcommunity` or `orb-community`?

Reply:
it's orbcommunity for docker hub and orb-community for github :)